src/main.c

```diff
@@ -102,7 +96,11 @@ static int		 output_description = 0;
 static int		 use_alternate_screen;
 static size_t		 query_size;
 static struct termios	 original_attributes;
-static struct choices	*choices;
+static struct {
+	size_t size;
+	size_t nmemb;
```

Comment:
Thanks! I can't come up with a good name for it. Is `v` for vector? Let's leave it be for now.